src/server/JasmineGraphInstanceService.cpp

```diff
@@ -472,6 +472,28 @@ void *instanceservicesession(void *dummyPt) {
             string result = "1";
             write(connFd, result.c_str(), result.size());
             instance_logger.log("Sent : " + result, "info");
+        } else if (line.compare(JasmineGraphInstanceProtocol::DELETE_GRAPH_FRAGMENT) == 0) {
+            /**
```

Comment:
Lets replace this multi-line comment with single line comment

Reply:
Replaced the multi-line comment comment with a single line comment under the else if condition line